### ISSUE TITLE
helper scripts for starting/closing

### ISSUE THREAD

**Issue by lachlancoin** [allenday/nanostream-dataflow#97]
I wrote some helper scripts:

https://github.com/lachlancoin/gcloud

These have a 'start.sh' which basically chains together all the steps for setup , provisioning and launch

and end.sh which does all the shutdown

Also, in the realtime directory is a script for watching a folder which is accumulating seq data and synching to the cloud bucket. 

Anyway, I ran the script and it seems to work ok, but anything I got wrong let me know.

**Comment by lachlancoin** [comment]
This is working well, but its spinning up two instances at startup, does that seem right?  I thought it would initially just have one instance

**Comment by lachlancoin** [comment]
tagging @obsh

**Comment by Firedrops** [comment]
Do the 2 instances persist? It was normal for me there to be multiple instances for a few minutes when starting up, but spin down after awhile and only leave 1 active.

**Comment by lachlancoin** [comment]
yes they persist.  once data put in pipe then 3 instances acitve

**Comment by lachlancoin** [comment]
Ok, so it seems that having 2 instances at the start maybe just for the minimap2 docker you did @Firedrops , and not for the BWA docker.
